data.sql

```diff
@@ -0,0 +1,35 @@
+INSERT INTO animals VALUES (
+  1,
+  'Agumon',
+  '2020-02-03',
+  0,
+  true,
+  10.23
+);
+
+INSERT INTO animals VALUES (
+  2,
+  'Gabumon',
+  '2018-11-15',
+  2,
+  true,
+  8.00
+);
+
+INSERT INTO animals VALUES (
+  3,
```

Comment:
Please specify the columns of table `animals` before inserting values so it will be definite which value is for which column.

Reply:
As per Postgres documentation, it isn't necessary to add column names if all the values are being added in the same order.